recipes/itkdb/recipe.yaml

```diff
@@ -0,0 +1,75 @@
+schema_version: 1
+
+context:
+  name: itkdb
+  version: "0.6.15"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/itkdb-${{ version }}.tar.gz
+  sha256: 4edfc211548e72499b12c3e6c4612a1d4ae4cbd13fafd22a136b2b638981cf28
+
+build:
+  skip:
+    - win
```

Comment:
Can this one be noarch?

It's okay if a noarch package fails on a specific platform due to a missing dependency. We prefer using noarch because it's faster to build, and the package will work as soon as the missing dependency becomes available on that platform.